website/content/docs/configuration/client.mdx

```diff
@@ -302,14 +302,22 @@ see the [drivers documentation](/nomad/docs/drivers).
   The default list is:
 
   ```text
-  CONSUL_TOKEN
```

Comment:
this isn't what Consul actually uses, but is it possible that folks might have it in their environment anyway?  I suppose admins could manually add it back in the config as described here, but it seems a little strange to open this door that's been shut all this time.

Reply:
Hmm, I guess this was either a typo or a maybe Consul used this env variable in the past. In any case, I will revert it, you're right.

Reply:
d1eb12d8b7adee2a80a46b0cfb55d5f897eab765

Reply:
I think it's a really common typo, because it's different from Nomad and Vault (and has been as long as I can remember):

* `NOMAD_ADDR` + `NOMAD_TOKEN`
* `VAULT_ADDR` + `VAULT_TOKEN`
* `CONSUL_HTTP_ADDR` + `CONSUL_HTTP_TOKEN`

:shrug: